packages/hyperion-autologging-visualizer/src/component/ALGraphInfo.react.tsx

```diff
@@ -51,6 +51,7 @@ const DefaultOptions: ALGraph.ALGraphDynamicOptionsType = {
     al_ui_event: {
       click: true,
       change: false,
+      hover: false,
```

Comment:
Increase the version number above from 1 to 2 to ensure everyone is forced to update the value.